include/cpplint_state.h

```diff
@@ -129,10 +129,10 @@ class CppLintState {
 
     // Print a summary of errors by category, and the total.
     void PrintErrorCounts();
-    void PrintInfo(const std::string& message);
-    void PrintError(const std::string& message);
+    void PrintInfo(const std::string& message) const;
+    void PrintError(const std::string& message) const;
 
-    bool AddJUnitFailure(const std::string& filename,
+    static bool AddJUnitFailure(const std::string& filename,
                          size_t linenum,
                          const std::string& message,
                          const std::string& category,
```

Comment:
You still need to align arguments for some functions that use `static` and `[[nodiscard]]`.